aws-mwaa-environment/aws-mwaa-environment.json

```diff
@@ -3,24 +3,6 @@
   "description": "Resource schema for AWS::MWAA::Environment",
   "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-mwaa.git",
   "definitions": {
-    "TagKey": {
```

Comment:
Do we know why the tag key and tag values are getting omitted?

Reply:
The `TagMap` definition was moved to L397-L400 under `Tags` for better compatibility with CDK.